source/_integrations/speedtestdotnet.markdown

```diff
@@ -1,50 +1,39 @@
 ---
-title: Speedtest.net
+title: SpeedTest.net
 description: How to integrate Speedtest.net within Home Assistant.
 ha_category:
   - System Monitor
   - Sensor
 ha_release: 0.13
 ha_iot_class: Cloud Polling
+ha_config_flow: true
 ha_codeowners:
   - '@rohankapoorcom'
+  - '@rngrbm87'
 ha_domain: speedtestdotnet
 ---
 
 The `speedtestdotnet` integration uses the [Speedtest.net](https://speedtest.net/) web service to measure network bandwidth performance.
 
-Enabling this integration will automatically create Speedtest.net sensors for the monitored conditions (below).
-
 By default, a speed test will be run every hour. The user can change the update frequency in the configuration by defining the `scan_interval` for a speed test to run.
 
 Most Speedtest.net servers require TCP port 8080 outbound to function. Without this port open you may experience significant delays or no results at all. See note on their [help page](https://www.speedtest.net/help).
 
 ## Configuration
 
-For the `server_id` check the list of [available servers](https://speedtest.net/speedtest-servers.php).
+Set up the integration through **Configuration -> Integrations -> SpeedTest.net**. Once configured you can select the server to run the test against from the options menu. You can also change the update interval and optionaly disable auto update.
```

Comment:
```suggestion
Set up the integration through **Configuration -> Integrations -> SpeedTest.net**. Once configured you can select the server to run the test against the options menu. You can also change the update interval and optionally disable auto-update.
```

Reply:
Here it is not "against the options menu". I meant to say "the server to run the test" against or with (not sure what the best term is) -- "from the options menu".

Reply:
Sorry, my bad... added a comma to make it read a bit better.